test/pipeline_test_runner.cc

```diff
@@ -754,13 +756,15 @@ TEST_CASE("PerPhaseTest") { // NOLINT
                     auto source = ctx.file.data(ctx).source();
                     parser::Prism::Parser prismParser{source};
                     bool collectComments = ctx.state.cacheSensitiveOptions.rbsEnabled;
```

Comment:
style nit: Space apart the "stages" of this a bit

 ```suggestion
 
                     bool collectComments = ctx.state.cacheSensitiveOptions.rbsEnabled;

```